src/components/AccordionMenu/index.spec.tsx

```diff
@@ -0,0 +1,163 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import { Store } from 'redux';
+
+import configureStore from '../../configureStore';
+import { actions } from '../../reducers/accordionMenu';
+import { createFakeEvent, shallowUntilTarget, spyOn } from '../../test-helpers';
+import styles from './styles.module.scss';
+
+import AccordionMenu, {
+  AccordionItem,
+  AccordionItemBase,
+  DefaultItemProps,
+  PublicItemProps,
+  makeItemContentId,
+} from '.';
+
+describe(__filename, () => {
+  describe('AccordionMenu', () => {
+    it('renders children', () => {
```

Comment:
The test is called `renders children`, but only one child is created. Do you think it would be better to have multiple children and assert that they all are rendered?

Reply:
In React, one child and multiple children are both accessed through `this.props.children` just the same -- there's no difference. There would be no additional code path to test.